schema_tests.py

```diff
@@ -0,0 +1,71 @@
+from underscore import _
+from application.deed.utils import validate_helper,\
+    call_once_only, get_obj_by_path
+from schema_tests.utils import load_json_file, is_valid_regex
+import os
+import sys
+
+
+@call_once_only
+def get_test_data():
+    return load_json_file(os.getcwd() +
+                          "/schema_tests/payload_tests/payload.json")
+
+
+@call_once_only
+def get_schema():
+    return load_json_file(os.getcwd() +
+                          get_obj_by_path(get_test_data(), "schema"))
+
+
+def verify_pattern(element, key, obj, idx):
+    print("Checking patterns for '%s'" % key)
```

Comment:
I have no strong feelings either way on this one, the logger both outputs into a file so in a kibana readable way so we could build a dashboard on our unittest passes over time and it outputs to the console so developers can see whats going on. I have no strong feelings either way other than consistency and history. If you think its a good idea, then it might be worth doing if not then no. Your decision.